packages/core-apis/console/yarn.lock

```diff
@@ -0,0 +1,2697 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I guess this file should be removed, right?

Reply:
yes only the root yarn.lock file should exist in the monorepo

Reply:
Oups :sweat_smile: